packages/block-library/src/media-text/media-container.native.js

```diff
@@ -84,9 +81,8 @@ class MediaContainer extends Component {
 	}
 
 	onUploadError( message ) {
-		const { noticeOperations } = this.props;
-		noticeOperations.removeAllNotices();
-		noticeOperations.createErrorNotice( message );
+		const { createErrorNotice } = this.props;
+		createErrorNotice( message );
```

Comment:
About this not working, this was actually copy-pasted from the web files and there's an [issue](https://github.com/wordpress-mobile/gutenberg-mobile/issues/2365) to remove this unused code but when we created that GH issue we didn't even have the `Notice` component 😅

I think we can remove `onUploadError` completely and trigger the `createErrorNotice` within `finishMediaUploadWithFailure` what do you think?

Reply:
> About this not working, this was actually copy-pasted from the web files and there's an issue to remove this unused code but when we created that GH issue we didn't even have the Notice component 😅

Understood! I never saw that issue 😅 . Thanks for pointing it out 🙇🏾 

> I think we can remove onUploadError completely and trigger the createErrorNotice within finishMediaUploadWithFailure what do you think?

I think that's a good idea. I will get that done.

Reply:
Resolved `onUploadError` behavior for both audio and media & text blocks here efac3de 9021214